app/controllers/tasks_controller.rb

```diff
@@ -2,12 +2,17 @@ class TasksController < ApplicationController
 
   def index
     @title = "Task List"
-    @tasks = Task.all
+    @tasks = Task.all.order(:completed_at)
```

Comment:
Great use of ActiveRecord's `order` method to get sorted results from the database!